banking/public/js/bank_reconciliation_beta/actions_panel/match_tab.js

```diff
@@ -473,11 +468,16 @@ erpnext.accounts.bank_reconciliation.MatchTab = class MatchTab {
 			{
 				name: __("Document Name"),
 				editable: false,
-				width: 100,
+				width: 150,
 				format: (value, row, column, data) => {
-					return frappe.format(value, {fieldtype: "Link", options: data[1]});
+					return frappe.format(value, {fieldtype: "Link", options: data[6]});
 				},
 			},
+			{
+				name: __("Document Type"),
+				editable: false,
+				width: 50,
+			},
```

Comment:
Could we even set this to hidden and only use it for formatting the link in Document Name? Usually the doctype should be obvious from the naming series.